### PR TITLE
Finalise the LVM tasks

### PR DESCRIPTION
**Pull request by KimForss**
## Problem
Update the LVM creation to support the new sap-parameters.yml file

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

## Notes
<Additional comments for the PR>

### CODE REVIEW

File: deploy/ansible/roles-os/1.5-disk-setup/tasks/main.yml [hunk 1]
```diff
@@ -97,31 +97,103 @@
...
   set_fact:
-    pvs_sap_str: "{{ pvs_sap | default([]) | join(',') }}"
+    pvs_backup_str: "{{ pvs_backup | default([]) | join(',') }}"
     cacheable: yes
 
-- name: Create the consolidated variable
+- name: Create the consolidated list
   set_fact:
-    LVM_data: "{{ [{ 'vg':'vg_sap', 'pvs': '{{ pvs_sap_str }}' },{ 'vg':'vg_data', 'pvs': '{{ pvs_data_str }}' },{ 'vg':'vg_log', 'pvs': '{{ pvs_log_str }}' }] }}"
+    LVM_data: "{{ [ \
```
Comment (rtamalin) [inline deploy/ansible/roles-os/1.5-disk-setup/tasks/main.yml:116]: You shouldn't need the `\` continuation characters on the ends of the line within the jinja expression.

Jinja will strip out any extraneous whitespace not explicitly inside quotes strings in the jinja expression so 

    "{{ [ { ... }, { ... } ] }}"

is equivalent to:

    "{{ [ \
          { ... }, \
          { ... } ] }}"

is equivalent to:

    "{{ [
          { ... },
          { ... } ] }}"

but

    "{{ [ 'abc
           def', ] }}"

is not equivalent to:

    "{{ [ 'abc\
           def', ] }}"

File: deploy/ansible/roles-os/1.5-disk-setup/tasks/main.yml [hunk 2]
```diff
@@ -97,31 +97,103 @@
...
+    { 'vg':'vg_backup',   'pvs': '{{ pvs_backup_str }}',  'lv':'lv_backup', 'size': '100%FREE', 'opt' : '',             'fstype': 'xfs', 'dev': '/dev/vg_backup/lv_backup'}, \
+    { 'vg':'vg_data',     'pvs': '{{ pvs_data_str }}',    'lv':'lv_data',   'size': '100%FREE', 'opt' : '-i 3 -I 128',  'fstype': 'xfs', 'dev': '/dev/vg_data/lv_data'}, \
+    { 'vg':'vg_log',      'pvs': '{{ pvs_log_str }}',     'lv':'lv_log',    'size': '100%FREE', 'opt' : '-i 3 -I 32',   'fstype': 'xfs', 'dev': '/dev/vg_log/lv_log'} \
```
Comment (hdamecharla) [inline deploy/ansible/roles-os/1.5-disk-setup/tasks/main.yml]: Hi @KimForss, according to our [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/hana-vm-operations-storage) the recommended stripe sizes are:
- 256 KB for /hana/data
- 64 KB for /hana/log

**Review by rtamalin — COMMENTED** [review]
Looks good.

**Review by hdamecharla — APPROVED** [review]
Hi @KimForss, approved #1118  👍🏾

**Pull request merged**